docs/secret_management.md

```diff
@@ -0,0 +1,172 @@
+# Secrets Management for the API and APP keys
+
+The Datadog-Operator is able to retrieves the datadog credentials securely thanks to 3 different methods.
```

Comment:
suggesting to revise to:
"Datadog Operator can be configured to retrieve the API and App Keys using secrets for enhanced security. There are three methods you can choose from to set it up:"

Reply:
it can also be used for the token exchange between the agent and cluster-agent.